perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -115,7 +118,7 @@
 	private SearcherBl searcherBl = null;
 	private TasksManagerBl tasksManagerBl = null;
 	private ConfigManagerBl configManagerBl = null;
-
+	private ConsentsManagerBl consentsManagerBl = null;
```

Comment:
should be probably added to toString() method also

Reply:
Done, I added missing ConfigManager as well.